test/e2e/network/dns.go

```diff
@@ -67,12 +66,40 @@ var _ = SIGDescribe("DNS", func() {
 		validateDNSResults(f, pod, append(wheezyFileNames, jessieFileNames...))
 	})
 
+
+	It("should resolve DNS of partial qualified names for the cluster ", func() {
+		// All the names we need to be able to resolve.
+		// TODO: Spin up a separate test service and test that dns works for that service.
+		namesToResolve := []string{
+			"kubernetes.default",
+			"kubernetes.default.svc",
+			"kubernetes.default.svc.cluster",
```

Comment:
I don't think kubernetes.default.svc.cluster is meant to be resolvable, since the ``resolv.conf`` file does not have the ``local`` dns suffix. The smallest suffix is ``cluster.local``.

Reply:
Removed the ``kubernetes.default.svc.cluster`` partial name, and did a few cleanups.